source/runner/templates/bitbucketCloudTemplate.ts

```diff
@@ -3,11 +3,11 @@ import { Violation } from "../../dsl/Violation"
 import { compliment } from "../DangerUtils"
 
 // BitBucket Cloud supports these emojis 🎉
-const noEntryEmoji = "❌"
-const warningEmoji = "⚠️"
-const messageEmoji = "✨"
+const noEntryEmoji = ":x:"
+const warningEmoji = ":warning:"
+const messageEmoji = ":sparkles:"
 const signatureEmoji = "🚫"
```

Comment:
Is there a specific reason why you did not update this one?

```suggestion
const signatureEmoji = ":no_entry_sign:"
```